packages/fxa-settings/.storybook/decorators.tsx

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { DecoratorFn } from '@storybook/react';
+import AppLocalizationProvider from 'fxa-react/lib/AppLocalizationProvider';
+
+// This decorator makes the localization bundles available in the stories.
+// If a localized string is available, that will be rendered in the storybook,
+// otherwise the fallback strings will be displayed.
+export const withLocalization: DecoratorFn = (Story) => (
+  <AppLocalizationProvider
+    baseDir="/locales"
+    bundles={['settings', 'react']}
+    userLocales={navigator.languages}
```

Comment:
`navigator.languages` is a standard property (https://developer.mozilla.org/en-US/docs/Web/API/Navigator/languages), and also what we are using elsewhere in the codebase for the `AppLocalizationProvider`, so I think this is correct.

Reply:
True, true; but for me locally in Firefox Nightly, my `navigator.languages` returns `[ "en-CA", "en-US", "en" ]`.
I wasn't sure if when testing in Storybook we wanted a bigger set of languages without needing to change browser settings.

Reply:
Ooooh, I see! That's an interesting thought. I hadn't consider if we could view different locales in the storybooks without changing browser settings. This add-on (https://storybook.js.org/addons/storybook-addon-locale) might be a good tool for that...  I'll investigate!

Reply:
One good language to test UIs in is German, because it is very... verbose.
But I imagine it could be tedious to have to change your browser config to quickly check that in Storybook.
Then again, this won't ever really work for freshly added strings since they won't have been translated yet, so I imagine trying to quickly test Storybook UIs like this won't be super optimal for an additional train or two after adding a new string.

---

I also fixed my `pontoonql` tool if we want to quickly see which locales are over n% translated:
```sh
npx pdehaan/pontoonql firefox-accounts 80
```

&hellip; which is pretty unpleasant output to read, but you can pipe it through `jq` or something to filter+format and then pipe further greps and count lines, etc.

```sh
npx pdehaan/pontoonql firefox-accounts 80 | jq '[.[] | {locale: .locale.code, progress: .progress, warnings: .stringsWithWarnings}]' | grep "locale" | wc -l # 38
```

Reply:
Had a look at the storybook add-ons - they're old and not actively maintained. Not sure it's worth the hassle of getting those to work for this PR. Not ideal, but I figure for the occasional time that we want to check in another language (say, if a bug is reported in German), we can use the browser settings to switch languages. Probably not too much hassle unless we want to check multiple languages, and I doubt that will be needed?

For the most part, I find that having l10n visible in the storybook is useful even with English as it lets us check that the localized strings are rendering properly and have all the necessary attributes.

---

Neat tool to check localization coverage! Thanks for sharing :)